config/config/src/index.ts

```diff
@@ -103,6 +103,7 @@ export const types = Object.assign({
   'package-import-method': ['auto', 'hardlink', 'clone', 'copy'],
   'patches-dir': String,
   pnpmfile: String,
+  'pm-strict': Boolean,
```

Comment:
I wouldn't abbreviate this personally. Not much else is abbreviated at the moment.

Reply:
ok

Reply:
renamed to `package-manager-strict`